src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$headerHeight: 60px;
```

Comment:
if you remember, today Anya said that header can be without height, 
links should set the height of the header 
(but I think it`s not important)